doc/changelogs/CHANGELOG_V010.md

```diff
@@ -0,0 +1,765 @@
+# Node.js v0.10 ChangeLog
+
+<table>
+<tr>
+<th colspan="2">Stable</th>
+</tr>
+<tr>
+<td valign="top">
+<a href="#0.10.44">0.10.44</a><br/>
+<a href="#0.10.43">0.10.43</a><br/>
+<a href="#0.10.42">0.10.42</a><br/>
+<a href="#0.10.41">0.10.41</a><br/>
+<a href="#0.10.40">0.10.40</a><br/>
+<a href="#0.10.39">0.10.39</a><br/>
+<a href="#0.10.38">0.10.38</a><br/>
+<a href="#0.10.37">0.10.37</a><br/>
+<a href="#0.10.36">0.10.36</a><br/>
+<a href="#0.10.35">0.10.35</a><br/>
+<a href="#0.10.34">0.10.34</a><br/>
+<a href="#0.10.33">0.10.33</a><br/>
+<a href="#0.10.32">0.10.32</a><br/>
+<a href="#0.10.31">0.10.31</a><br/>
+<a href="#0.10.30">0.10.30</a><br/>
+<a href="#0.10.29">0.10.29</a><br/>
+<a href="#0.10.28">0.10.28</a><br/>
+<a href="#0.10.27">0.10.27</a><br/>
+<a href="#0.10.26">0.10.26</a><br/>
+<a href="#0.10.25">0.10.25</a><br/>
+<a href="#0.10.24">0.10.24</a><br/>
+<a href="#0.10.23">0.10.23</a><br/>
+<a href="#0.10.22">0.10.22</a><br/>
+</td>
+<td valign="top">
+<a href="#0.10.21">0.10.21</a><br/>
+<a href="#0.10.20">0.10.20</a><br/>
+<a href="#0.10.19">0.10.19</a><br/>
+<a href="#0.10.18">0.10.18</a><br/>
+<a href="#0.10.17">0.10.17</a><br/>
+<a href="#0.10.16">0.10.16</a><br/>
+<a href="#0.10.15">0.10.15</a><br/>
+<a href="#0.10.14">0.10.14</a><br/>
+<a href="#0.10.13">0.10.13</a><br/>
+<a href="#0.10.12">0.10.12</a><br/>
+<a href="#0.10.11">0.10.11</a><br/>
+<a href="#0.10.10">0.10.10</a><br/>
+<a href="#0.10.9">0.10.9</a><br/>
+<a href="#0.10.8">0.10.8</a><br/>
+<a href="#0.10.7">0.10.7</a><br/>
+<a href="#0.10.6">0.10.6</a><br/>
+<a href="#0.10.5">0.10.5</a><br/>
+<a href="#0.10.4">0.10.4</a><br/>
+<a href="#0.10.3">0.10.3</a><br/>
+<a href="#0.10.2">0.10.2</a><br/>
+<a href="#0.10.1">0.10.1</a><br/>
+<a href="#0.10.0">0.10.0</a><br/>
+</td>
+</tr>
+</table>
+
+* Other Versions
```

Comment:
This is what I was referring to before, that will be broken in release streams.

I don't think we need to include the links to the other change logs... thoughts?